lib/app.jsx

```diff
@@ -114,7 +114,12 @@ export const App = connect( mapStateToProps, mapDispatchToProps )( React.createC
 
 	onAppCommand: function( command ) {
 		if ( command != null && typeof command === 'object' && command.action != null && this.props.actions.hasOwnProperty( command.action ) ) {
-			this.props.actions[ command.action ]( command );
+			// Special case for newNote since it needs the note bucket prop from this class
```

Comment:
No, sorry if I wasn't clear. If you look at the (newNote)[https://github.com/Automattic/simplenote-electron/blob/issue/249-keyboard-shortcuts/lib/flux/app-state.js#L261] action, it requires that the bucket object gets passed to it. These `appCommands` usually call the action methods directly which is why I added this case.